lib/serve.ls

```diff
@@ -8,7 +8,7 @@ require! {
   './build.ls'
 }
 
-reflex-path = path.dirname require.resolve 'reflex/server'
+reflex-path = path.dirname path.dirname require.resolve 'reflex/lib/server'
```

Comment:
probably better to resolve `reflex/package.json` as that will always sit in the root directory